webui/react/src/components/Logo.module.scss

```diff
@@ -15,3 +15,6 @@
     width: 190px;
   }
 }
+.customLogo {
+  max-width: 200px;
+}
```

Comment:
does this need to be only applied conditionally when there's a custom logo? seems like it could just be part of a `.base` class that's always applied.

Reply:
would this make sense ? https://github.com/determined-ai/determined/pull/9664#discussion_r1686823575